### PR TITLE
Navigation Updated

### PR DESCRIPTION
**Pull request by CharlyCee**
Navigation reviewed and Updated.

### CODE REVIEW

File: front-end/src/components/Navigation.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+import { AppBar, Toolbar, Link, Box, Stack } from "@mui/material";
+import Avatar from "@mui/material/Avatar";
 
 const pages = [
 	{ name: "HOME", link: "/" },
 	{ name: "COURSES", link: "/courses" },
 ];
-
-function Navigation() {
+function Navigation({ isLogin = true }) {
```
Comment (BorisLaboratory) [inline front-end/src/components/Navigation.js:9]: I am not sure why you did { isLogin = true }. I think you meant Navigation({ isLogin }). ie  can be false or true from Where Navigation component is invoked

**Review by BorisLaboratory — CHANGES_REQUESTED** [review]
im not sure if you have considered acceptance craterial#2 ie to add more space before login link. Also I think Navigation should be passed {isLogin} without specifying true or false.Every other thing looks good to me. Good work

File: front-end/src/components/Navigation.js (hunk 1 above)
Comment (CharlyCee) [reply front-end/src/components/Navigation.js:9]: For isLogin, Delvin suggested I leave it to be true.

Comment (CharlyCee) [reply front-end/src/components/Navigation.js:9]: Additional space will be reviewed and added. Prettier reformed it. 
Thank you

**Review by dellydel — DISMISSED** [review]
The default of true is fine for now since the authentication piece is not yet in place. We can update it when a user can actually log in

**Review by BorisLaboratory — APPROVED** [review]

**Review by dellydel — APPROVED** [review]

**Pull request merged**